waddrmgr/address.go

```diff
@@ -534,6 +534,14 @@ func newManagedAddressFromExtKey(s *ScopedKeyManager,
 	return managedAddr, nil
 }
 
+// clearTextSetter is a non-exported interface to identify script types that
+// allow their clear text script to be set.
+type clearTextSetter interface {
```

Comment:
Clear text here is just a top-level identifier that isn't the hashed script?

Reply:
It's just an internal optimization (to avoid decrypting the script multiple times) that was previously done through a direct struct type assert and is now done through a private interface. Updated the name to make it a bit more clear what exactly is being set.